lambdas/drivingpermitcheck/src/test/java/uk/gov/di/ipv/cri/drivingpermit/api/handler/DrivingPermitHandlerTest.java

```diff
@@ -130,9 +132,9 @@ void handleResponseShouldReturnOkResponseWhenValidInputProvided()
                 .sendAuditEvent(eq(AuditEventType.RESPONSE_RECEIVED), any(AuditEventContext.class));
 
         // Choose API
-        when(mockConfigurationService.getUseLegacy()).thenReturn(true);
+        when(mockConfigurationService.getDvaDirectEnabled()).thenReturn(true);
```

Comment:
Should this not now be false for this test?

Reply:
You're right. Corrected.